chain/indexer/src/streamer/mod.rs

```diff
@@ -84,30 +94,32 @@ async fn build_streamer_message(
         let mut chunk_receipts = chunk_local_receipts;
         chunk_receipts.extend(chunk_non_local_receipts);
 
+        // Add local receipts to corresponding outcomes
+        for receipt in &local_receipts {
+            if let Some(outcome_position) = outcomes
+                .iter()
+                .position(|outcome| outcome.execution_outcome.id == receipt.receipt_id)
```

Comment:
Use [`find`](https://doc.rust-lang.org/stable/std/iter/trait.Iterator.html#method.find) instead of `position`, to get the element itself instead of the position, so you don't need to `get` + `expect`.

Reply:
find doesn't return mutable, and I need to mutate it

Reply:
ah, sorry `iter_mut` did the trick